forge-gui/res/cardsfolder/upcoming/invasion_of_zendikar__awakened_skyclave.txt

```diff
@@ -0,0 +1,8 @@
+Name:Invasion of Zendikar
```

Comment:
I had the full version earlier but it was causing problems that i thought were related to the alternate mode